tools/scripts/build_ubuntu_x64_ncnn.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+import time
+
+from ubuntu_utils import cmd_result, ensure_base_env
+
+g_jobs = 2
```

Comment:
Can we make it larger? The build process on host is very slow

Reply:
done.